src/databricks/labs/ucx/source_code/base.py

```diff
@@ -41,25 +42,6 @@ class Advice:
     end_line: int
     end_col: int
 
-    def replace(
```

Comment:
what about directfs accesses?

Reply:
already done in https://github.com/databrickslabs/ucx/pull/2599 and https://github.com/databrickslabs/ucx/pull/2630

Reply:
what is left re directfs is more focused methods that call dataclasses.replace